iStore/NetworkLayer/ProductDetailsView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  ProductDetailsView.swift
+//  iStore
+//
+//  Created by Mohamed Sayed on 14.02.24.
+//
+
+import SwiftUI
+
+struct ProductDetailsView: View {
+    
+    @StateObject var productDetailsViewModel = ProductDetailsViewModel(networkManager: NetworkManager())
+    
+    @State private var isFavoriteTapped = false
+    var id: Int
+    
+    var body: some View {
+        Group {
+            if productDetailsViewModel.isloading {
+                ProgressView()
+                    .frame(maxWidth: .infinity, maxHeight: .infinity)
+                    .background(.black)
+                    .tint(.white)
+                    .ignoresSafeArea()
+            } else {
+                VStack() {
```

Comment:
please try to divide the `VStack` to subviews

Reply:
ok